wdl/tasks/PBUtils.wdl

```diff
@@ -47,16 +50,23 @@ task GetRunInfo {
     input {
         String subread_bam
 
+        String? bam_suffix
+
         RuntimeAttr? runtime_attr_override
     }
 
     String gcs_dir = sub(subread_bam, basename(subread_bam), "")
 
+    String bam_suffix_arg = if defined(bam_suffix) then " --BS " else ""
+
     command <<<
         set -x
 
         export GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token`
-        python /usr/local/bin/detect_run_info.py ~{gcs_dir} > run_info.txt
+
+        # We need to update detect_run_info.py to make it sanitize fields.
+        # The `sed` statement here is a hack to get around an issue.
+        python /usr/local/bin/detect_run_info.py ~{gcs_dir} ~{bam_suffix_arg}~{default="" sep=" --BS " bam_suffix} | sed 's#\\$##' > run_info.txt
```

Comment:
Let's just update the detect_run_info.py script.  Would be cleaner.

Reply:
Sounds good.  Done!